docs/howto.rst

```diff
@@ -4,18 +4,6 @@ How to
 
 .. contents::
 
-Connect to PostgreSQL Database in local dev environment
-=======================================================
-
-The local development environment's PostgreSQL database exposes itself on a
-non-standard port when run with docker-compose. You can connect to it with
-the client of your choice using the following connection settings:
-
-* Username: ``postgres``
-* Password: ``aPassword``
-* Port: ``8574``
```

Comment:
I moved this to the local dev chapter where it's relevant.

Reply:
Also, the "processing pii access requests" section should get moved to the runbook and removed from these docs. I'll do that next.